examples/cosmwasm/counter/models/counter.tla

```diff
@@ -114,8 +110,7 @@ Next==
     /\ messages' = Append(messages, last_msg)
     /\ stepCount' = stepCount + 1
 
-Inv == stepCount < MAX_STEP
+MaxLength == stepCount < MAX_STEP
```

Comment:
Are these kind of invariants necessary, now that we have simulation mode from Apalache?

Reply:
Hm. so you're saying we would mostly use `simulate` mode rather than `sample` and `check`?